lib/lib-openapi/src/test/java/org/sagebionetworks/translator/ControllerModelDocletTest.java

```diff
@@ -262,4 +263,52 @@ public void testDescriptionandContentForVoidReturnMethodWithNoRedirect() {
 		assertEquals("The request was successful, but there is no response content.", okObj.getString("description"));
 		assertThrows(JSONException.class, () -> okObj.getString("content"));
 	}
+
+	@Test
+	public void testHttpServletResponseNotIncludedAsParameter() {
+		JSONObject pathsObj = generatedOpenAPISpec.getJSONObject("paths");
+		JSONObject pathObj = pathsObj.getJSONObject("/repo/v1/complex-pet/file/{fileId}/url");
```

Comment:
method path can include information about why this method was added for testing.